runtime/ddr/vmddrstructs.properties

```diff
@@ -238,6 +238,8 @@ ddrblob.typeoverride.J9ROMNameAndSignature.signature=J9SRP(struct J9UTF8)
 
 ddrblob.typeoverride.J9ROMMethodTypeRef.signature=J9SRP(struct J9UTF8)
 
+ddrblob.typeoverride.J9ROMConstantDynamicRef.nameAndSignature=J9SRP(struct J9ROMNameAndSignature)
```

Comment:
Should this not also be added to `runtime/ddr/overrides-vm`?

Reply:
Yes, this was missed. I've created https://github.com/eclipse/openj9/pull/11706 to address this.